python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -211,7 +211,11 @@ def __init__(self, registry: Registry, args: SimpleNamespace, writeable: bool):
                     raise ValueError("Cannot use --extend-run option with non-existing or empty output chain")
                 runName = self.output.chain[0]
             else:
-                runName = "{}/{}".format(self.output, Instrument.makeCollectionTimestamp())
+                import random
+
+                extra = str(random.randint(0, 9))
```

Comment:
This is a temporary hack to guarantee that the code hits the race condition where we roll over to the next second in the timestamp calculation.